### PR TITLE
feat: providing divider SCSS function

### PR DESCRIPTION
**Pull request by mfranzke**
To enable borders with a transparent color related to the current `color` used on that element we need to provide a workaround previous to Relative Color Syntax being available in the relevant browsers.

### CODE REVIEW

File: packages/foundations/scss/helpers/_functions.scss [hunk 1]
```diff
@@ -35,3 +35,38 @@
...
+		background-color: currentColor;
+		opacity: 0.16;
+
+		position: absolute;
+		@if $position == "top" {
+			height: 1px;
+			top: 0;
+			right: 0;
+			left: 0;
+		} @else if $direction == "bottom" {
```
Comment (nmerget) [inline packages/foundations/scss/helpers/_functions.scss:55]: Why do you use `$direction` here?

File: packages/foundations/scss/helpers/_functions.scss [hunk 2]
```diff
@@ -35,3 +35,38 @@
...
+			height: 1px;
+			bottom: 0;
+			right: 0;
+			left: 0;
+		} @else if $direction == "left" {
+			width: 1px;
+			bottom: 0;
+			top: 0;
+			left: 0;
+		} @else if $direction == "left" {
```
Comment (nmerget) [inline packages/foundations/scss/helpers/_functions.scss:65]: Should this be `right`? :P

**Comment by nmerget** [comment]
Can you add this to the DBDivider Component as well? So we have the same behavior an opacity etc.

File: packages/foundations/scss/helpers/_functions.scss [hunk 3]
```diff
@@ -35,3 +35,38 @@
...
+// https://codepen.io/bramus/pen/VwMZOQK
+// https://bugs.chromium.org/p/chromium/issues/detail?id=1274133
+// https://github.com/WebKit/WebKit/commit/1890bfb551da24735804cb1a3d5f14b210f7c7b8
+@mixin divider($position: "top", $slot: "before") {
+	&::#{$slot} {
+		content: "";
+		background-color: currentColor;
+		opacity: 0.16;
+
+		position: absolute;
```
[the remark this answers is not in the document]
Comment (mfranzke) [reply packages/foundations/scss/helpers/_functions.scss:51]: I've added that one for resilience of the workaround into the mixin as well.

File: packages/foundations/scss/helpers/_functions.scss (hunk 1 above)
Comment (mfranzke) [reply packages/foundations/scss/helpers/_functions.scss:57]: You're my Maximilian in code reviews.

**Comment by mfranzke** [comment]
> Can you add this to the DBDivider Component as well? So we have the same behavior an opacity etc.

Divider behaves totally differently (regarding how the SCSS mixin is setup to fit those needs for borders).

**Review by nmerget — APPROVED** [review]

**Pull request merged**